gcp/eck/eck-yamls/entsearch-fleet-kibana.yaml

```diff
@@ -21,7 +21,6 @@ spec:
             memory: ${kibana_pod_memory}
             cpu: ${kibana_pod_cpu}
   config:
-    xpack.fleet.agents.elasticsearch.hosts: ["https://eck-elasticsearch-es-http.default.svc:9200"]
```

Comment:
was told I need to remove this to get it working

Reply:
@aag6z  can you verify this config was tested?

Reply:
@sunileman I tested it - it works